app/controllers/bookings_controller.rb

```diff
@@ -1,2 +1,54 @@
 class BookingsController < ApplicationController
+  before_action :set_band
+
+  def index
+    @bookings = Booking.all
```

Comment:
If you want a band or an organiser to only see his bookings
if current_user.is_band
@bookings = Booking.where("band_id = #{@band.id}")
elsif current_user.is_organiser
@bookings = Booking.where("organiser_id = #{@organiser_id}")
end